woocommerce/woocommerce-ios
# Fix 5.8 metadata

This PR fixes the 5.8 metadata issues cause by the `download_metadata.swift` script deleting the whole `metadata` directory each time.

Note: The Apple TV privacy policy text is all missing, but we don't have an app for that platform, so I don't mind just leaving it blank for now to avoid noise in future diffs

**To Test:**
- Run `cd fastlane && ./download_metadata.swift`. Ensure that the repo is clean.
